modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,29 +1,50 @@
-= Release Notes for Couchbase Service Broker 1.0
+= Release Notes for Couchbase Service Broker {service-broker-version-minor}
+include::partial$constants.adoc[]
 
-Couchbase Service Broker 1.0 provides the first native integration of Couchbase Autonomous Operator with Open Service Broker API for the Open Source Kubernetes platform.
+The Couchbase Service Broker provides native integration of the Couchbase Autonomous Operator with the https://www.openservicebrokerapi.org/[Open Service Broker API^] for Open Source Kubernetes .
 
-Couchbase Service Broker 1.0 provides a service class named _couchbase-osb-service_ with two service plans: `csb-basic` and `csb-standard` (with the regular Couchbase Server Enterprise subscription), that will work with any version of Couchbase Autonomous Operator. However, we recommend using Couchbase Autonomous Operator version 2.1.
+Couchbase Service Broker {service-broker-version-minor} provides a service class named _couchbase-osb-service_ with two service plans: `csb-basic` and `csb-standard` (with the regular Couchbase Server Enterprise subscription), that will work with any version of Couchbase Autonomous Operator.
```

Comment:
Actually it provides:
* class: couchbase-autonomous-operator
  * plan: 2.1.0
  * plan: 2.2.0
* class: cao-2.1-service
  * plan: csb-basic
  * plan: csb-standard
* class: cao-2.2-service
  * plan: csb-basic
  * plan: csb-standard

The operator class is not intended to be used directly, but is instead instantiated by the cao plans.  This maintains a separation between couchbase CR's and the operator itself.  Why you may ask, well these things are API compatible, and having then separate also addresses independent upgrades.

Additionally these service classes are provided on a per-namespace, or global basis.

So where we used to have a nice loose coupling, we now have a very much tighter one, just to make documentation a lot harder :smile_cat: